src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -1141,6 +1151,20 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
                         );
                         this.disconnectDelayId = res.delay_id;
                     } catch (e) {
+                        if (
+                            e instanceof MatrixError &&
+                            e.errcode === "M_UNKNOWN" &&
```

Comment:
MSC says it should be M_INVALID_PARAM https://github.com/matrix-org/matrix-spec-proposals/blob/toger5/expiring-events-keep-alive/proposals/4140-delayed-events-futures.md#m_invalid_param-instead-of-m_max_delay_exceeded

Reply:
oh hecking unstable prefixes, this is awful